src/devices/Mcp25xxx/Mcp25xxx.csproj

```diff
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <EnableDefaultItems>false</EnableDefaultItems>
+    <LangVersion>latest</LangVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="System.Device.Gpio" Version="0.1.0-prerelease*" />
+    <Compile Include="Mcp25xxx.cs" />
+    <Compile Include="Mcp2515.cs" />
+    <Compile Include="Mcp25625.cs" />
+    <Compile Include="ReadStatusResponse.cs" />
+    <Compile Include="RxBufferAddressPointer.cs" />
+    <Compile Include="RxStatusResponse.cs" />
+    <Compile Include="TxBufferAddressPointer.cs" />
+    <Compile Include="Register\*.cs" />
```

Comment:
would it make sense to change these to: `Register\**\*.cs`?

Reply:
Yes, makes sense.  Will update.